rdrf/rdrf/admin.py

```diff
@@ -490,7 +490,7 @@ class CDEFileAdmin(admin.ModelAdmin):
     ADMIN_COMPONENTS = ADMIN_COMPONENTS + DESIGN_MODE_ADMIN_COMPONENTS
 
 if settings.SYSTEM_ROLE is SystemRoles.NORMAL:
-    ADMIN_COMPONENTS = ADMIN_COMPONENTS + NORMAL_MODE_ADMIN_COMPONENTS
+    ADMIN_COMPONENTS = ADMIN_COMPONENTS + NORMAL_MODE_ADMIN_COMPONENTS + PROMS_ADMIN_COMPONENTS
```

Comment:
Why is this needed? We would never have PROM links in NORMAL mode.

Reply:
It's a leftover before I changed url resolving using reverse_lazy. Removed the change.